### DISCUSSION TITLE
Manually change networks?

### DISCUSSION THREAD

**Discussion by CorrodedAura** [RandomNinjaAtk/docker-sonarr-extended#28] [Q&A]
Is there a way to manually change the network to YouTube for the Youtube Series Downloader?

**Comment by RandomNinjaAtk** [comment]
I'm not sure I follow, but everything is determined by the TVDB data, so if you want to change something, you'll need to change on that site (https://thetvdb.com/)

**Reply by CorrodedAura** [reply]
I'm just trying to import a web series that's also hosted elsewhere to pull down from YT automatically. There are also several mini-episode/extra things that are on YT that would be nice to be able to grab separately.

**Reply by RandomNinjaAtk** [reply]
If you can provide examples of series on the TVDB that are directly linked to the episode on a different site and work with yt-dlp, I can look into it.

**Reply by CorrodedAura** [reply]
https://thetvdb.com/series/rwby-fairy-tales
This is the specific example I'm dealing with right now, though the modern run of Dr Who (for the minisodes and extras) also runs into the same issue, that being most of it's on youtube only. With RWBY, the Network is set to rooster teeth, but the show also exist on youtube as well and it would be nice to be able to have an option to pull it from there (honestly, this might just be a problem with Sonarr as a whole)

**Reply by RandomNinjaAtk** [reply]
None of the episodes are linked to the actual content... So it's not possible to pull the content...

The script requires the actual episodes to be linked to the actual site, here is an example:
Bad (not linked episode): https://thetvdb.com/series/rwby-fairy-tales/episodes/8810843
Good (linked episode):  https://thetvdb.com/series/the-slow-mo-guys/episodes/9567241

See this issue for more details: https://github.com/RandomNinjaAtk/docker-sonarr-extended/issues/21
